backend/src/main/resources/application-create-sample-data.yaml

```diff
@@ -59,4 +59,47 @@ simple-report-initialization:
       manufacturer: Access Bio, Inc.
       model: CareStart COVID-19 Antigen test*
       loinc-code: "94558-4"
-      swab-type: "258500001"
\ No newline at end of file
+      swab-type: "258500001"
+simple-report:
+  site-admin-emails:
+    - ruby@example.com
+  demo-users:
+    default-user:
+      identity:
+        username: bob@sample.com
+        first-name: Bob
+        last-name: Bobberoo
+      authorization:
+        organization-external-id: ${SR_DEMO_USER_ORG:DIS_ORG}
+        granted-roles: ${SR_DEMO_USER_ROLE:ADMIN}
+    alternate-users:
+      - identity:
```

Comment:
for my own edification, am i understanding this correctly that `findAllOrganizationRoles()` just returns an empty list for Ruby now, and that's tolerated in the app?

Reply:
That is correct—that is now the default behavior for site-wide admin users in production, as well. (We previously had a dummy organization so that we weren't constantly unintentionally downloading customer PHI—making Organization nullable in `whoami` was actually sneakily intended to support this use case.)